pkg/operator/ceph/cluster/cluster.go

```diff
@@ -599,30 +599,41 @@ func (c *cluster) reportTelemetry() {
 func (c *cluster) configureMsgr2() error {
 	encryptionSetting := "secure"
 	defaultRBDMapOptions := "ms_mode=secure"
+	rbdMapOptions := "rbd_default_map_options"
 	encryptionGlobalConfigSettings := map[string]string{
-		"ms_cluster_mode":         encryptionSetting,
-		"ms_service_mode":         encryptionSetting,
-		"ms_client_mode":          encryptionSetting,
-		"rbd_default_map_options": defaultRBDMapOptions,
+		"ms_cluster_mode": encryptionSetting,
+		"ms_service_mode": encryptionSetting,
+		"ms_client_mode":  encryptionSetting,
+		rbdMapOptions:     defaultRBDMapOptions,
```

Comment:
Might want to get rid of `defaultRBDMapOptions` variable too.  It's used in only one place and "default" in the name is confusing: is the default to enable just msgr2 or msgr2 + encryption?
```suggestion
		rbdMapOptions:     "ms_mode=secure",
```

Reply:
`defaultRBDMapOptions` is actually a good name for the variable that you are introducing here...